model.py

```diff
@@ -177,7 +177,7 @@ def from_json(cls, json_dict):
                 id=json_dict.get('_id', None))
 
 class Tournament(object):
-    def __init__(self, type, raw, date, name, players, matches, regions, orig_ids=None, id=None):
+    def __init__(self, type, raw, url, date, name, players, matches, regions, orig_ids=None, id=None):
```

Comment:
bleh, so there is a bit of a problem with this: now everywhere we initialize a Tournament (in dao or elsewhere), we need to update this argument (because of this tons of tests are failing). 

i think the solution is to: a) actually update all of these references b) add url as an optional named parameter at the end (i.e. put url="" or url=None at the end of the list of parameters).

eventually we need to clean up how these models work (which is what the ORM refactor was intended to do).